test/index.js

```diff
@@ -1125,7 +1125,7 @@ let AllTests = ContextureClient => {
     expect(targetTree.getNode(['root', 'a']).values).to.deep.equal([])
 
     // Mutate on sourceTree will await the Subquery into targetTree
-    expect(spy).to.have.callCount(2)
+    expect(spy).to.have.callCount(1)
```

Comment:
The change on this PR infects this subquery test.  the second time mutate did not change the value of snapshot. since we have smart mutate that mutate should not trigger a dispatch event